src/test/java/com/education/service/ClassServiceTest.java

```diff
@@ -4,97 +4,88 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.transaction.annotation.Transactional;
+import org.springframework.test.context.jdbc.Sql;
 
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.AFTER_TEST_METHOD;
 
+@Sql(executionPhase = AFTER_TEST_METHOD, value = "/sql/cleanup.sql")
 @SpringBootTest
-@Transactional
 public class ClassServiceTest {
+
     @Autowired
     private ClassService classService;
 
     @Test
     public void save() {
-        Class room = new Class();
-        room.setClassNumber("125");
-        room.setCapacity(45);
-        classService.save(room);
-        Long roomId = room.getId();
+        Class class1 = SampleBuilder.class1();
+
+        classService.save(class1);
 
-        Class roomById = classService.getById(roomId);
+        Class ClassById = classService.getById(class1.getId());
 
-        assertNotNull(roomById);
-        assertEquals("125", roomById.getClassNumber());
-        assertEquals(45, roomById.getCapacity());
+        assertNotNull(ClassById);
+        assertEquals("111", ClassById.getClassNumber());
+        assertEquals(20, ClassById.getCapacity());
     }
 
     @Test
     public void getById() {
-        Class room = new Class();
-        room.setClassNumber("123");
-        room.setCapacity(12);
-        classService.save(room);
-        Long roomId = room.getId();
+        Class class1 = SampleBuilder.class1();
 
-        Class roomById = classService.getById(roomId);
+        classService.save(class1);
 
-        assertNotNull(roomById);
-        assertEquals("123", roomById.getClassNumber());
-        assertEquals(12, roomById.getCapacity());
+        Class classById = classService.getById(class1.getId());
+
+        assertNotNull(classById);
+        assertEquals("111", classById.getClassNumber());
+        assertEquals(20, classById.getCapacity());
     }
 
     @Test
     public void delete() {
-        Class room = new Class();
-        room.setClassNumber("111");
-        room.setCapacity(20);
-        classService.save(room);
-        Long roomId = room.getId();
+        Class class1 = SampleBuilder.class1();
```

Comment:
why class1?
I think 1 is misleading.
Also, method name should be verb

Reply:
can I consider createClass1?